internal/driver/interactive.go

```diff
@@ -87,6 +97,13 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 						o.UI.PrintErr(err)
 					}
 					continue
+				} else {
+					if okValues := groups[name]; okValues != nil {
+						sort.Strings(okValues)
+						okValuesStr := strings.Join(okValues, ", ")
+						o.UI.PrintErr(fmt.Errorf("Unrecognized value for %s: %q. Use one of %s", name, value, okValuesStr))
```

Comment:
Nit: maybe just use `strings.Join(okValues, ", ")` instead of okValuesStr here, it's not much longer.

Reply:
Done.